requirements/dev

```diff
@@ -0,0 +1,25 @@
+# Please make sure to pin all dependency versions, in order to avoid unwanted
+# functional and integration breaks caused by external code updates.
+#
+# These are dev dependencies, so pin them.
```

Comment:
> Unnecessary considering the above comments.

Reply:
I improved the comment, please let me know if that's OK.